content/yaml/building-a-native-ios-app.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Building a native iOS app
+description: Building an iOS app with YAML.
+weight: 4
+---
+
+With `codemagic.yaml`, you can use Codemagic to build, test and publish native iOS apps. You can read more about how to use codemagic.yaml and see the structure of the file [here](../yaml/yaml).
+
+## Building a native iOS app
+
+For building an iOS app, you need to run the following command in the scripts section:
+
+    - xcode-project build-ipa --project "MyXcodeProject.xcodeproj" --scheme "MyScheme"
+
+## Testing an iOS app
+
+The code for testing an iOS app also goes under `scripts`. The relevant code for a native iOS app looks like this:
+
+    xcodebuild \
+    -workspace MyAwesomeApp.xcworkspace \
+    -scheme MyAwesomeApp \
+    -sdk iphonesimulator \
+    -destination 'platform=iOS Simulator,name=iPhone 6,OS=8.1' \
+    test | xcpretty (edited) 
+
+More examples of testing with YAML can be found [here](../yaml/testing).
+
+## Code signing
```

Comment:
The same comment here: there's not enough information to have a separate section, it makes more sense to add code signing and publishing information as bullet points or in a notebox